### PR TITLE
chore: Moves the stylesheets folder to the assets folder

### PR DESCRIPTION
**Pull request by michael-s-molina**
### SUMMARY
Moves the `stylesheets` folder to the `assets` folder.

This work is part of [SIP-61](https://github.com/apache/superset/issues/13632)

@jinghua-qa @junlincc 

### TESTING INSTRUCTIONS
1 - Navigate through the application
2 - Check that all styles are loaded correctly

### ADDITIONAL INFORMATION
- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/package.json [hunk 1]
```diff
@@ -19,10 +19,10 @@
     "build-instrumented": "cross-env NODE_ENV=development BABEL_ENV=instrumented webpack --mode=development --color",
     "build": "cross-env NODE_OPTIONS=--max_old_space_size=8192 NODE_ENV=production BABEL_ENV=\"${BABEL_ENV:=production}\" webpack --mode=production --color",
     "lint": "eslint --ignore-path=.eslintignore --ext .js,.jsx,.ts,.tsx . && npm run type",
-    "prettier-check": "prettier --check '{src,stylesheets}/**/*.{css,less,sass,scss}'",
+    "prettier-check": "prettier --check '{src}/**/*.{css,less,sass,scss}'",
     "lint-fix": "eslint --fix --ignore-path=.eslintignore --ext .js,.jsx,.ts,tsx . && npm run clean-css && npm run type",
-    "clean-css": "prettier --write '{src,stylesheets}/**/*.{css,less,sass,scss}'",
-    "format": "prettier --write './{src,spec,stylesheets,cypress-base}/**/*{.js,.jsx,.ts,.tsx,.css,.less,.scss,.sass}'",
+    "clean-css": "prettier --write '{src}/**/*.{css,less,sass,scss}'",
+    "format": "prettier --write './{src,spec,cypress-base}/**/*{.js,.jsx,.ts,.tsx,.css,.less,.scss,.sass}'",
```
Comment (villebro) [inline superset-frontend/package.json:25]: The cases with `'{src}/**/...'` would probably look nicer as just `'src/**/...'`

**Review by villebro — COMMENTED** [review]
LGTM - not sure why the linter is unable to find the stylesheets based on the updated globs as they look totally fine to me?

File: superset-frontend/package.json (hunk 1 above)
Comment (michael-s-molina) [reply superset-frontend/package.json]: That was the thing making the linter fail. I changed to your suggestion.

**Comment by jinghua-qa** [comment]
I have tested in the ephemeral environment and i saw one issue:

When i run query in sql, i did not see 'EXPLORE' option under results tab and did not see this issue in master,  not sure whether is this PR related. Can you check that?

Screen Shot 2021-09-29 at 10 52 55 AM

**Comment by michael-s-molina** [comment]
@jinghua-qa Thanks for testing! I was able to reproduce the same problem on master so it's not related to this PR. I checked and it's not related to the stylesheets. I took note here and I will fix the problem in another PR.

**Comment by michael-s-molina** [comment]
@jinghua-qa The problem with the Explore button was fixed here https://github.com/apache/superset/pull/16908

**Review by pkdotson — APPROVED** [review]
LGTM! thanks for the changes.

**Pull request merged**